packages/apps/plugins/plugin-navtree/src/components/NavTreeContainer.tsx

```diff
@@ -103,16 +95,14 @@ export const NavTreeContainer = ({
     await dispatch({
       action: NavigationAction.OPEN,
       data: {
-        // TODO(thure): don’t bake this in, refactor this to be a generalized approach.
```

Comment:
I think this comment was in reference to the comments popping open vs the deck model

Reply:
I think it's legacy from when we did a whole bunch of logic here in manually splicing the active parts.

Reply:
Maybe it's referring to the string formatting

Reply:
@ZaymonFC my recollection was that we didn't want the navtree to know specifically about comments and that there should be a generalized approach for "related planks" to automatically be opened.